src/tests/test_davidson.cc

```diff
@@ -178,10 +332,12 @@ class BlockOperator : public MatrixFreeOperator {
   int _diag;
```

Comment:
do we use that ?

Reply:
Ahh that was the thing it always was complaining about.